client/dashboard/utils/site-name.ts

```diff
@@ -0,0 +1,6 @@
+import { getSiteUrlWithoutProtocol } from './site-url';
```

Comment:
It's current "pattern" to have individual "utils" file for different parts of the site, instead of a single util file. 😅